spec.html

```diff
@@ -45,7 +45,7 @@ <h1>JSON.isRawJSON ( _O_ )</h1>
   <emu-clause id="sec-json.parse">
     <h1>JSON.parse ( _text_ [ , _reviver_ ] )</h1>
     <p>The `parse` function parses a JSON text (a JSON-formatted String) and produces an ECMAScript language value. The JSON format represents literals, arrays, and objects with a syntax similar to the syntax for ECMAScript literals, Array Initializers, and Object Initializers. After parsing, JSON objects are realized as ECMAScript objects. JSON arrays are realized as ECMAScript Array instances. JSON strings, numbers, booleans, and null are realized as ECMAScript Strings, Numbers, Booleans, and *null*.</p>
-    <p>The optional _reviver_ parameter is a function that takes <del>two</del><ins>three</ins> parameters, _key_ and _value_ <ins>and a _context_ object</ins>. It can filter and transform the results. It is called with each of the _key_/_value_ pairs produced by the parse <ins>and (for primitive values) details of their corresponding Parse Node</ins>, and its return value is used instead of the original value. If it returns what it received, the structure is not modified. If it returns *undefined* then the property is deleted from the result.</p>
+    <p>The optional _reviver_ parameter is a function that takes <del>two</del><ins>three</ins> parameters, _key_, _value_, <ins>and a _context_ object</ins>. It can filter and transform the results. It is called with each of the _key_/_value_ pairs produced by the parse <ins>and (for primitive values) details of their corresponding Parse Node</ins>, and its return value is used instead of the original value. If it returns what it received, the structure is not modified. If it returns *undefined* then the property is deleted from the result.</p>
```

Comment:
This invalidates the ins/del diff because those commas are not there now, but we _can_ be a little more radical.
```suggestion
    <p>The optional _reviver_ parameter is a function that <del>takes two parameters, _key_ and _value_. It</del> can filter and transform the results. <del>It is called with each of the _key_/_value_ pairs produced by the parse,</del><ins>For each value produced by the parse, it is called with three arguments (the key, the value, and a context object containing [for primitive values] details of the corresponding Parse Node)</ins> and its return value is used instead of the original value. If it returns what it received, the structure is not modified. If it returns *undefined* then the property is deleted from the result.</p>
```